src/main/scad/yavuz-demirhan/Raya1.scad

```diff
@@ -0,0 +1,66 @@
+include <puzzlecad2.scad>
+
+box_puzzle_scale = 15;
+box_puzzle_border = 6;
+$burr_inset = 0.16; 
+long = box_puzzle_scale * 4; // voxels long
+wide = box_puzzle_scale * 3; // voxels wide
+high = box_puzzle_scale * 2; // voxels high
+
+dimlong = long + box_puzzle_border * 2;
+dimwide = wide + box_puzzle_border * 2;
+height = high + box_puzzle_border;
+
+mid = box_puzzle_border / 2 + 0.5;
+far = dimlong - mid;
+far2 = dimwide - mid;
+
+pieces();
+*box();
+*cap();
+
+module pieces() {
+    burr_plate([
+    ["xx|xx"],  //piece A
+    ["x..|xxx"],//piece B
+    ], $burr_scale = box_puzzle_scale,$plate_width = 150,$burr_bevel = 1.0);
+}
+
+module box() {
+    
+    difference() {
+        beveled_cube([dimwide, dimlong, height]);
+        translate([box_puzzle_border - $burr_inset, box_puzzle_border - $burr_inset, box_puzzle_border- $burr_inset])
+        cube([
+            wide + $burr_inset * 2,
+            long + $burr_inset * 2,
+            high + $burr_inset * 1]);
```

Comment:
I suggest adding an additional $burr_inset to the height (so that there is an added tolerance of $burr_inset * 2 along each dimension). This may be why you had to increase the inset (see above)?